custom/enikshay/tasks.py

```diff
@@ -397,6 +397,14 @@ def _get_fulfilled_vouchers(self):
         ]
         return sorted(relevant_vouchers, key=self._get_fulfilled_voucher_date)
 
+    def _get_fulfilled_available_vouchers(self):
+        relevant_vouchers = [
+            voucher for voucher in self._get_all_vouchers()
+            if (voucher.get_case_property('voucher_type') == 'prescription'
+                and voucher.get_case_property('state') in ['fulfilled', 'available'])
```

Comment:
Hmm, I'm wondering if it is better if we added those as a "payment_status" property? Otherwise, I've made that change here: https://github.com/dimagi/commcare-hq/pull/16411